### PR TITLE
raftstore: skip handling read only command

### PR DESCRIPTION
**Pull request by BusyJay**
Read only command are handled in raftstore directly,  so we don't need to do anything in apply worker.

### CODE REVIEW

**Comment by javaforfun** [comment]
LGTM

File: src/raftstore/store/worker/apply.rs [hunk 1]
```diff
@@ -875,10 +873,13 @@ impl ApplyDelegate {
         for req in requests {
             let cmd_type = req.get_cmd_type();
             let mut resp = try!(match cmd_type {
-                CmdType::Get => self.handle_get(ctx, req),
                 CmdType::Put => self.handle_put(ctx, req),
                 CmdType::Delete => self.handle_delete(ctx, req),
-                CmdType::Snap => self.handle_snap(ctx, req),
+                // Readonly command are handled in raftstore directly.
```
Comment (siddontang) [inline src/raftstore/store/worker/apply.rs]: commands are

File: src/raftstore/store/worker/apply.rs [hunk 2]
```diff
@@ -875,10 +873,13 @@ impl ApplyDelegate {
...
+                // Don't panic here in case there are old entries need to be applied.
+                // It's also safe to skip them here, because a restart must have happened,
+                // hence there is no callback to be called.
+                CmdType::Snap | CmdType::Get => continue,
```
Comment (siddontang) [inline src/raftstore/store/worker/apply.rs]: should log a warning here

**Comment by siddontang** [comment]
Rest LGTM

Btw, CI failed

**Pull request merged**